tests/serialization/testdata/serialize_by_file/TestDigestSerializer/deep_model_folder

```diff
@@ -1 +1 @@
-36eed9389ebbbe15ac15d33c81dabb60ccb7c945ff641d78f59db9aa9dc47ac9
+6c4326b2e7cb2a08c96523093b89302e219e8c29ac0d6c2fe6d9803191794001
```

Comment:
Oh, I was expecting these to not change, but probably I'm wrong here. Need to investigate

Reply:
the merge hasher header no longer factors in directories

Reply:
Ohh, that explains it. Thank you!